google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -633,16 +641,16 @@ private static String getPomVersion(String artifactId) {
     return null;
   }
 
-  private static String getManifestVersion() {
+  private String getManifestVersion() {
     String version = null;
     try {
       Enumeration<URL> resources =
           ServiceOptions.class.getClassLoader().getResources(JarFile.MANIFEST_NAME);
       while (resources.hasMoreElements() && version == null) {
         Manifest manifest = new Manifest(resources.nextElement().openStream());
         Attributes manifestAttributes = manifest.getMainAttributes();
-        String artifactId = manifestAttributes.getValue(MANIFEST_ARTIFACT_ID_KEY);
-        if (artifactId != null && artifactId.equals(ARTIFACT_ID)) {
+        String currentArtifactId = manifestAttributes.getValue(MANIFEST_ARTIFACT_ID_KEY);
```

Comment:
What does "current" mean here? This code is in google-cloud-core, which implies this is "core artifact id".